### PR TITLE
Update system-requirements.md

### PR DESCRIPTION
**Pull request by joeabbey**
Discovered while investigating https://github.com/docker/docker-core-backlog/issues/446

### CODE REVIEW

File: ee/ucp/admin/install/system-requirements.md [hunk 1]
```diff
@@ -94,7 +94,20 @@ Number 4 for IP-in-IP encapsulation.
 
 If you're deploying to AWS or another cloud provider, enable IP-in-IP
 traffic for your cloud provider's security group.
- 
+
+## Enable connection tracking on the loopback interface for SLES
+Calico's Kubernetes controllers can't reach the Kubernetes API server
+unless connection tracking is enabled on the loopback interface.  (SLES
```
Comment (fcrisciani) [inline ee/ucp/admin/install/system-requirements.md]: SLES12 SP2

**Comment by traci-morrison** [comment]
@joeabbey Hi, Joe. We are working through our backlog. Can you confirm if this PR is still valid? Thanks.

**Comment by traci-morrison** [comment]
@joeabbey PTAL. Are these changes still applicable or can this PR be closed? Please advise. Thanks.

**Comment by traci-morrison** [comment]
@KoryKessel-Docker PTAL. I resolved the merge conflict, but made no additional changes.

**Pull request merged**